README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://github.com/dart-lang/jnigen/workflows/Dart%20CI/badge.svg)](https://github.com/dart-lang/jnigen/actions?query=workflow%3A"Dart+CI")
```

Comment:
I think maybe you want to filter to main?
https://github.com/dart-lang/jnigen/actions?query=workflow%3A%22Dart+CI%22+branch%3Amain

Reply:
done